starter_code/express-spotify-app/views/albums.ejs

```diff
@@ -0,0 +1,11 @@
+<h2 class="big-bad-title">Learn about Music</h2>
+
+<div class="band-container">
+        <% for (let i = 0; i < 3; i++) { %>
+        <div class="band">
+            <h3 class="artist-title"><%= albums[i].name %></h3>
+            <img class="album-cover" src="<%= albums[i].images[0].url %>"></img>
```

Comment:
The best approach is define a helper function that, if there is an image, return it, otherwise displays a placeholder. You can use that function in artists.ejs too.